backend/src/model/entity/plants_impl.rs

```diff
@@ -6,10 +6,7 @@ use diesel_async::{AsyncPgConnection, RunQueryDsl};
 use crate::db::function::array_to_string;
 use crate::db::pagination::Paginate;
 use crate::model::dto::{Page, PageParameters, PlantsSearchParameters};
-use crate::{
```

Comment:
Probably better to put this into a separate PR to not cause conflicts with #217.

Reply:
I'll leave it in for now. If there are conflicts I'll create a separate PR. Its not that much work anyways.